xbmc/windowing/egl/EGLNativeTypeAmlogic.h

```diff
@@ -53,6 +53,8 @@ class CEGLNativeTypeAmlogic : public CEGLNativeType
 protected:
   bool SetDisplayResolution(const char *resolution);
   void SetupVideoScaling(const char *mode);
+  void DealWithScale(const RESOLUTION_INFO &res);
```

Comment:
`EnableFreeScale` and `DisableFreeScale` can be private as well. I made it protected to match `DisableFreeScale`. Should I move all three to private?